www/js/app.js

```diff
@@ -1016,12 +1017,14 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             setLocalArchiveFromArchiveList();
         }
         else {
```

Comment:
Might as well standardize this to `} else {` as well.

Reply:
Yes, missed it somehow. will do